.prettierrc

```
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "all"
```

Maybe you need add this, so you wouldn't have problem with quotes.
```suggestion
  "trailingComma": "all"
  "singleQuote": "true"
```